src/pauli.jl

```diff
@@ -0,0 +1,229 @@
+module pauli
+
+export PauliBasis, PauliTransferMatrix, ChiMatrix
+
+import Base: ==
+
+using ..bases, ..spin, ..superoperators
+using ..operators: identityoperator, AbstractOperator
+using ..superoperators: SuperOperator
+using ..operators_dense: DenseOperator
+using ..spin: sigmax, sigmay, sigmaz
+using SparseArrays
+using LinearAlgebra: tr
+
+"""
+    PauliBasis(num_qubits::Int)
+
+Basis for an N-qubit space where `num_qubits` specifies the number of qubits.
+The dimension of the basis is 2²ᴺ.
+"""
+mutable struct PauliBasis{B<:Tuple{Vararg{Basis}}} <: Basis
+    shape::Vector{Int}
+    bases::B
+    function PauliBasis(num_qubits::Int64)
```

Comment:
Use `::Int` here (and at other points in the file) in order for things to work properly on Win32

Reply:
great